### PR TITLE
Replace use of plivo sdk by requests

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: pip-requires.txt [hunk 1]
```diff
@@ -18,12 +18,11 @@ dj-database-url
 smartmin
 
 ########################## services ##########################
-analytics-python==0.4.4
```
Comment (rowanseymour) [inline pip-requires.txt]: you need to update this against master? this was from a different PR by dodo no?

Comment (norkans7) [reply pip-requires.txt]: right, the change was made by hand in pip freeze, this make sure pip-tools compile the same version

**Comment by rowanseymour** [comment]
@norkans7 I didn't know "LGTM" before I saw it in one of your comments, but LGTM once you fix the conflicts. Also have we tested this?

**Comment by nicpottier** [comment]
Can we ship this to staging to test it out?

**Comment by norkans7** [comment]
I pushed in on staging and this is GTG

**Pull request merged**